Update_VISOR_Linux.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+git checkout -f main
+git reset --hard origin/main
+git pull origin main
+python./tools/download_models.py
```

Comment:
Probably you should remove this line and the line below? The files don't exist on the repo

Reply:
oof...fixing it.